newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesListController.php

```diff
@@ -141,17 +144,13 @@ public function getPlaylistsArticlesAction(Request $request, $id)
         }
 
         $playlistArticles = $em->getRepository('Newscoop\Entity\Playlist')
-            ->articles($playlist, null, false, null, null, $onlyPublished, true)->getResult();
+            ->articles($playlist, array(), true, null, null, $onlyPublished, true)->getResult();
 
-        $articlesIds = array();
+        $articles = array();
         foreach ($playlistArticles as $playlistArticle) {
-            $articlesIds[] = $playlistArticle['articleId'];
+            $articles[] = $playlistArticle->getArticle();
```

Comment:
This will break. You are setting $fullArticle in the method _articles()_ to true, which means it will do: _$query->select('p, a')_. This returns an array mixed with object of \Newscoop\Entity\Article and \Newscoop\Entity\PlaylistArticle.